pkgs/top-level/all-packages.nix

```diff
@@ -5500,6 +5500,8 @@ with pkgs;
 
   whois = callPackage ../tools/networking/whois { };
 
+  wireguard-tools = callPackage ../tools/networking/wireguard-tools { };
```

Comment:
Mostly to avoid ambiguity, reduce potential confusion and make it clear that when you are installing `wireguard-tools` you are getting only tools, not all of WireGuard. Also because `wireguard-tools` are a separate package in many other distros.

Reply:
I would prefer keep the upstream name instead because it is easier to guess.

Reply:
Also this renaming affect our nixos module.

Reply:
I believe `wireguard-tools` is actually easier to guess, because it is called this way in (almost) every other major distro (Debian, Fedora, Arch, OpenSUSE, OpenWrt, Homebrew). So this name is consistent with the way others name it and, more importantly, with what the package actually does.
I guess, let’s wait for what @zx2c4 has to say on this matter.

Reply:
> Also the reference in our nixos module needs to be renamed.

I believe I renamed it right in my first commit.